repos/c2sm/packages/makedepf90/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Makedepf90(Package):
```

Comment:
https://salsa.debian.org/science-team/makedepf90/-/blob/debian/latest/configure?ref_type=heads#L3 suggests this is an autotools package. Is it not?